sw/device/tests/sim_dv/lc_ctrl_transition_test.c

```diff
@@ -75,7 +74,9 @@ bool test_main(void) {
     dif_lc_ctrl_token_t token;
     dif_lc_ctrl_settings_t settings;
     settings.clock_select = kDifLcCtrlInternalClockEn;
-    memcpy(token.data, lc_exit_token, sizeof(lc_exit_token));
+    for (int i = 0; i < LC_TOKEN_SIZE; i++) {
```

Comment:
The previous code should still work, right? Any reason for changing it?

Reply:
There is a compile error if we use `memcpy` on `volatile` array. But if we remove the `volatile`, the SV backdoor write won't work.